### PR TITLE
Feat/deliverable1

### PR DESCRIPTION
**Pull request by enriquedfa**
Needs lots of visual improvements :( but I got the idea at least.

### CODE REVIEW

**Comment by enriquedfa** [comment]
Needs lots of visual improvements :( but I got the idea at least.

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,75 @@
     transform: rotate(360deg);
   }
 }
+
+/* MY CSS */
+.header {
```
Comment (danbenitezf) [inline src/App.css:41]: I think you can divide this css section into different ones, one for each its corresponding component. Here you have the header,  footer, featured products, etc. It's better to divide so you only modify the one you need and easier to identify where the style comes from

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import Header from './components/header/Header';
+import Footer from './components/footer/Footer';
+import Homepage from './components/homepage/Homepage';
 
 function App() {
   const { data, isLoading } = useFeaturedBanners();
```
Comment (danbenitezf) [inline src/App.js:8]: As far as I see, i think this hook is not being used here so better to remove it. Also try to not leave any console when making the push

File: src/components/header/Header.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { MdShoppingCart, MdSearch } from "react-icons/md"
+
+function Header() {
+    return (
+        <header className="header">
+            <div className="header__logo">
+                E Commerce
+                <a href="/">
+                    {/* <img src="https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png" alt="E Commerce" /> */}
```
Comment (danbenitezf) [inline src/components/header/Header.js:10]: Commented blocks of code are not usually left, it's better to remove them

File: src/components/homepage/Categories.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import categories  from "./product-categories.json"
+
+function Categories() {
```
Comment (danbenitezf) [inline src/components/homepage/Categories.js:4]: Try to review this component, it's not showing correctly on your netlify preview. I'm also adding a general comment for the PR regarding the naming and structure of the folders to make them more organized and easier to understand the code structure

File: src/components/homepage/Featured.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import featured from "./featured-products.json"
+
+// Returns a grid of featured products
+function Featured() {
+    return (
+        <div className="featured">
+            <h2>Featured Products</h2>
+            <div className="featured-products">
+                {featured.results.map(product => (
```
Comment (danbenitezf) [inline src/components/homepage/Featured.js:10]: Something you can change here is to destructure the results content for example 
`const { results } = featured;` so this way then you do the map it's only
` results.map( product => () );`

**Review by danbenitezf — APPROVED** [review]
Good job and don't worry about the styles, what matters the most is that the criteria is being met. 
It would be better if you take another approach of organizing the folders.
image
I've renamed some of your files and reorganized the project structure, it's not obligatory to change them, but as you can see the it's easier to identify every component. Also the mock files are saved on their own folder.

Take a look into naming convention for react and the use of `.jsx` extension vs `.js`
Also your categories are now showing correctly, not at least on netlify. Take a look at that and don't worry if right now that criteria is not met as it can be changed later. 
image

## Evaluation criteria:

- [x] The Header is rendered correctly (25 points)
- [x] The Featured Banners Slider is rendered correctly using the data from the mock file (20 points)
- [] The Categories Carousel/Grid is rendered correctly using the data from the mock file (20 points)
- [x] The Featured Products Grid is rendered correctly using the data from the mock file (30 points)
- [x] The Footer is rendered correctly (5 points)
- [ ] BONUS: Unit Test to validate that the Ecommerce Store logo/name appears correctly in the Header. (10 points)

If you believe of the evaluation criterias is wrong don't hesitate to ask a review

File: src/App.css (hunk 1 above)
Comment (enriquedfa) [reply src/App.css:41]: All right, I though about the same too, I will split the styles by components and import them on the main CSS

File: src/components/header/Header.js (hunk 3 above)
Comment (enriquedfa) [reply src/components/header/Header.js:10]: Wups, forgot to remove some commented code.

**Comment by enriquedfa** [comment]
Thanks for the review! I will try to fix and improve all the things mentioned in your comments on another PR.

**Pull request merged**